dashboard/app/models/resource.rb

```diff
@@ -97,9 +97,20 @@ def summarize_for_lesson_edit
     }
   end
 
+  def summarize_for_script_unit_group_overview
```

Comment:
nit: The name of this method is confusing to me. Does this mean it is summarizing for both the script overview and unit group overview?

Reply:
I had the same confusion. I think `summarize_for_script_or_unit_group_overview` or just `summarize_for_overview` would make more sense to me

Reply:
or `summarize_for_teacher_resources_dropdown`

Reply:
> or `summarize_for_teacher_resources_dropdown`

that sounds better!

Reply:
Yes, I like it. Thanks!